setuptools/config/_validate_pyproject/__init__.py

```diff
@@ -5,7 +5,7 @@
 from .error_reporting import detailed_errors, ValidationError
 from .extra_validations import EXTRA_VALIDATIONS
 from .fastjsonschema_exceptions import JsonSchemaException, JsonSchemaValueException
-from .fastjsonschema_validations import validate as _validate
+from .fastjsonschema_validations import validate as _validate  # type: ignore[attr-defined] # mypy false-positive. Pyright is fine here
```

Comment:
`setuptools/config/_validate_pyproject` should also be ignored because it is automatically generated using an external tool...

Reply:
Thanks for the heads-up, I missed that. Since `attr-defined` is an issue that comes up often for lots of different reasons that are not gonna be an easy fix in setuptools (and should not be a priority focus), I've disabled that error code globally.